packages/contracts-bedrock/src/libraries/Transient.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+/// @title Transient
+/// @notice Library for transient storage.
+library Transient {
+    /// @notice Slot for call depth.
+    ///         Equal to bytes32(uint256(keccak256("transient.calldepth")) - 1).
+    uint256 internal constant CALL_DEPTH_SLOT = 0x7a74fd168763fd280eaec3bcd2fd62d0e795027adc8183a693c497a7c2b10b5c;
+
+    /// @notice Gets the current call depth.
+    /// @return _callDepth Current call depth.
+    function getCallDepth() internal view returns (uint256 _callDepth) {
```

Comment:
Can we rename things a bit to be more concise? 

`library Transient` isn't super clear about what it does which means we need to make the function names super long. If we named the library `TransientContext` then we can simplify the names of the functions to `get()`, `increment()` and `decrement()` which i think will make the library much cleaner

Reply:
`function callDepth()(uint256)`